modules/billing/payment-required/data.js

```diff
@@ -0,0 +1,103 @@
+import { useEffect } from 'react'
+import { autorun, observable } from 'mobx'
+import { useObserver } from 'mobx-react-lite'
+
+import { requestPremium as letter } from 'texts/letters'
+
+const globalContext = observable({
+  user: {},
+  organisation: {},
+})
+
+const userInput = observable({
+  update(patch) {
+    Object.assign(this, patch)
+  },
+})
+
+const emailContext = observable({
+  toEmail: 'enterp\u0072is\u0065\u0040cor\u0065\u002e\u0061\u0063\u002euk',
```

Comment:
To prevent SPAM scanners detect and collect email from the JS code or from source code (that is why it's here obfuscated too). I am not sure it's needed, @samuelpearce used to do so, I think it's not bad to protect.

Despite it leaks to the HTML in the source look, I do not mind it in the closed project. However, we may add some protection there too.

Reply:
Interesting, that's quite clever.